revdep/problems.md

```diff
@@ -0,0 +1,150 @@
+# httpuv
+
+<details>
+
+* Version: 1.6.1
+* GitHub: https://github.com/rstudio/httpuv
+* Source code: https://github.com/cran/httpuv
+* Date/Publication: 2021-05-07 04:50:02 UTC
+* Number of recursive dependencies: 38
+
+Run `revdep_details(, "httpuv")` for more info
+
+</details>
+
+## In both
+
+*   checking for GNU extensions in Makefiles ... NOTE
+    ```
+    GNU make is a SystemRequirements.
+    ```
+
+# plumber
+
+<details>
+
+* Version: 1.1.0
+* GitHub: https://github.com/rstudio/plumber
+* Source code: https://github.com/cran/plumber
+* Date/Publication: 2021-03-24 05:10:02 UTC
+* Number of recursive dependencies: 76
+
+Run `revdep_details(, "plumber")` for more info
+
+</details>
+
+## In both
+
+*   checking tests ...
+    ```
+     ERROR
+    Running the tests in ‘tests/testthat.R’ failed.
+    Last 13 lines of output:
+      <simpleError in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]): namespace 'htmltools' 0.5.1.1 is already loaded, but >= 0.5.1.9004 is required>
```

Comment:
Is this something we need to deal with?

Reply:
I'll double check. Prolly a half dev / half check installation 😞

Reply:
Fixed!